include/swift/AST/DiagnosticEngine.h

```diff
@@ -425,25 +439,72 @@ namespace swift {
     /// Add a character-based range to the currently-active diagnostic.
     InFlightDiagnostic &highlightChars(SourceLoc Start, SourceLoc End);
 
+    static const char *fixItStringFor(const FixItID id);
+
+    /// Add a token-based replacement fix-it to the currently-active
+    /// diagnostic.
+    template <typename... ArgTypes>
+    InFlightDiagnostic &
+    fixItReplace(SourceRange R, StructuredFixIt<ArgTypes...> fixIt,
+                 typename detail::PassArgument<ArgTypes>::type... VArgs) {
+      DiagnosticArgument DiagArgs[] = { std::move(VArgs)... };
+      return fixItReplace(R, fixItStringFor(fixIt.ID), std::move(DiagArgs));
```

Comment:
Nitpick: No `std::move` needed here (though it shouldn't be harmful), since the parameter type is a reference to an existing array.

Reply:
must've missed that while refactoring, fixed!